src/Reindex.php

```diff
@@ -31,17 +29,11 @@ class Reindex extends Param
     public const REFRESH_WAIT_FOR = 'wait_for';
     public const WAIT_FOR_COMPLETION = 'wait_for_completion';
 
-    /**
-     * @deprecated since version 7.2.0, use a boolean as parameter instead.
-     */
-    public const WAIT_FOR_COMPLETION_FALSE = 'false';
     public const WAIT_FOR_ACTIVE_SHARDS = 'wait_for_active_shards';
     public const TIMEOUT = 'timeout';
     public const SCROLL = 'scroll';
     public const REQUESTS_PER_SECOND = 'requests_per_second';
     public const PIPELINE = 'pipeline';
-    public const SLICES = 'slices';
-    public const SLICES_AUTO = 'auto';
```

Comment:
We should leave those two consts, as they can be defined in the Reindex request:
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html

Reply:
@thePanz I've undone the removal.

Reply:
I guess the `setSlices(int $slices)` method is missing, which would then use those constants
Please leave them, I will provide a PR to use them later :)

Reply:
LGTM, would you please fix the CS warnings?
Then I can merge it
\cc @ruflin